resources/assets/coffee/react/beatmapset-page/scoreboard-table-row.coffee

```diff
@@ -53,34 +56,49 @@ export class ScoreboardTableRow extends React.PureComponent
 
       td className: cell,
         a
-          className: "#{bn}__user-link js-usercard"
+          className: "#{bn}__cell-content #{bn}__cell-content--user-link js-usercard"
           'data-user-id': score.user.id
           href: laroute.route 'users.show', user: score.user.id, mode: @props.beatmap.mode
           score.user.username
 
-      td className: osu.classWithModifiers(cell, ['perfect'] if score.max_combo == @props.beatmap.max_combo),
+      el @tdLink,
+        modifiers: ['perfect'] if score.max_combo == @props.beatmap.max_combo
         "#{osu.formatNumber(score.max_combo)}x"
 
       for stat in @props.hitTypeMapping
-        td
+        el @tdLink,
           key: stat[0]
-          className: osu.classWithModifiers(cell, ['zero'] if score.statistics["count_#{stat[1]}"] == 0),
+          modifiers: ['zero'] if score.statistics["count_#{stat[1]}"] == 0
           osu.formatNumber(score.statistics["count_#{stat[1]}"])
 
-      td className: osu.classWithModifiers(cell, ['zero'] if score.statistics.count_miss == 0),
+      el @tdLink,
+        modifiers: ['zero'] if score.statistics.count_miss == 0
         osu.formatNumber(score.statistics.count_miss)
 
-      td className: cell, _.round score.pp
+      el @tdLink,
+        {}
+        round score.pp
 
-      td className: osu.classWithModifiers(cell, ['time']),
+      el @tdLink,
+        modifiers: ['time']
         el ScoreboardTime,
           dateTime: score.created_at
 
-      td className: osu.classWithModifiers(cell, ['mods']),
+      el @tdLink,
```

Comment:
doesn't quite work at all for this column

![](https://s.myconan.net/2020-11/msedge_2020-11-20_16-05-09.png)